frappe/commands/site.py

```diff
@@ -660,16 +660,16 @@ def uninstall(context, app, dry_run, yes, no_backup, force):
 
 @click.command('drop-site')
 @click.argument('site')
-@click.option('--root-login', default='root')
-@click.option('--root-password')
+@click.option('--mariadb-root-username', default='root', help='Root username for MariaDB')
+@click.option('--mariadb-root-password', help='Root password for MariaDB')
```

Comment:
Shouldn't the options be DB agnostic? Also, we shouldn't introduce breaking changes to the CLI; perhaps keep the old options as fallback/aliases.

Reply:
Yes. It should be DB agnostic. 

I am not sure if we have db agnostic support yet https://github.com/frappe/frappe/blob/develop/frappe/database/__init__.py#L18

If we have db agnostic support, we can rename/add options for all other commands as well.